cmd/installer/pkg/install/install.go

```diff
@@ -48,7 +49,9 @@ func Install(p runtime.Platform, seq runtime.Sequence, opts *Options) (err error
 	cmdline.SetAll(p.KernelArgs().Strings())
 
 	// first defaults, then extra kernel args to allow extra kernel args to override defaults
-	cmdline.AppendDefaults()
+	if err = cmdline.AppendAll(config.DefaultKernelArgs()); err != nil {
```

Comment:
we should be using default cmdline args also in QEMU/Firecracker provisioners

Reply:
might be easier to pull in go-procfs without default args and see where things break :)

Reply:
Well, yeah. I guess we can do that, probably need to change `Append` to `Set` for whatever flags are added in the provider.

Reply:
oh, I see. There's also NewDefaultCmdline... Ok, I guess I'll update go-procfs now

Reply:
Should be good to go now.